utils/clients/icat_client.py

```diff
@@ -48,8 +48,8 @@ def _test_connection(self):
         """
         try:
             self.client.refresh()
-        except icat.exception.ICATSessionError:
-            raise ConnectionException("ICAT")
+        except icat.exception.ICATSessionError as exc:
```

Comment:
This is very minor but I noticed that sometimes you use `as exp` and other times `as exc`